src/arch/arm/machine/debug.c

```diff
@@ -941,8 +431,10 @@ static int getAndResetActiveBreakpoint(word_t vaddr, word_t reason)
              * range, which means it's not guaranteed to match the aligned value
              * that was programmed into the address register.
              */
-            align_mask = convertArchToSize(dbg_bcr_get_byteAddressSelect(bcr));
+#ifdef CONFIG_ARCH_AARCH32
```

Comment:
If you put the ifdef around the comment above, it would be clearer that it only applies to 32-bit. I assume aarch64 has support for unaligned values in the address register?